src/ledger_api.rs

```diff
@@ -0,0 +1,123 @@
+use ledger::*;
+
+use super::*;
+
+use helium_api::Client;
+
+use super::traits::B58;
+use crate::result::Result;
+use keypair::PubKeyBin;
+use prettytable::Table;
+
+const INS_GET_PUBLIC_KEY: u8 = 0x02;
+const INS_SIGN_PAYMENT_TXN: u8 = 0x08;
+
+enum PubkeyDisplay {
```

Comment:
Future devs might appreciate comments explaining this enum's purpose. I get it from the name, but I don't understand when you'd want to display or not display the public key, or where it's displayed. But, looking at the usage below, it appears to be on hardware. In that case, what about

```rust
#[repr(u8)]
enum PubkeyDisplay {
    Off = 0,
    On = 1,
 }
```

then using it as is (line 24 below):

```rust
         p1: display as u8,
```